src/util/StringUtils.h

```diff
@@ -207,6 +207,33 @@ inline size_t findLiteralEnd(const std::string_view input,
   return endPos;
 }
 
+// Implementation based on https://stackoverflow.com/a/25374036
+// Note because of ambiguous overload resolution the function name
+// here has the 'Impl' suffix, even though the explicit conversion is required
+// for it to compile (so only one overload actually works in practice)
+inline bool constantTimeEqualsImpl(std::basic_string_view<volatile char> str1,
+                                   std::basic_string_view<volatile char> str2) {
+  if (str1.length() != str2.length()) {
+    return false;
+  }
```

Comment:
Yes, if the access token is sufficiently long this isn't an issue at all. Of course if the access token is just a few characters long this could be considered a weakness, but in this case a simple brute force attack is more likely.